pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -939,7 +1041,7 @@ def cmd_instance_count(context, classname, options):
         except CIMError as ce:
             # allow processing to continue if no interop namespace
             if ce.status_code == CIM_ERR_NOT_FOUND:
-                click.echo('WARNING: {}'.format(ce))
+                warning_msg('{}'.format(ce))
```

Comment:
This change is unrelated to the purpose of this PR. Please move to a separate PR.
Occurs multiple times in this file.

Reply:
DONE